mongodb-odbc-driver/bin/create-dmg.sh

```diff
@@ -22,7 +22,9 @@
 
 	# copy driver libraries to appropriate location
 	cp "$DRIVERS_DIR"/*.so ./
-
+	# copy openssl to appropriate location
+	cp "/usr/local/Cellar/openssl/1.0.2r/lib/libcrypto.1.0.0.dylib" ./
+	cp "/usr/local/Cellar/openssl/1.0.2r/lib/libssl.1.0.0.dylib" ./
```

Comment:
we're using the path `/usr/local/Cellar/openssl/1.0.2r/` in a bunch of different scripts. Perhaps we could factor it out into `prepare-shell.sh` and call it `$BREW_OPENSSL_PATH` or similar?

Reply:
Yea, the thing is it's used across 3 projects, but I guess I can at least clean it up here.